### DISCUSSION TITLE
How to use aliases without having to source .bashrc every time I open a new terminal?

### DISCUSSION THREAD

**Discussion by jrobinson0529** [nss-evening-cohort-14/evening-client-side#14] [Look at what I learned!]
#### [Here](https://wiki.manjaro.org/index.php/Aliases_in_.bashrc) is a wiki with more information on bash and aliases if you want to set up your own.

-------------------------------------------------------
What I initially searched first:

[First Search](https://superuser.com/questions/130192/how-do-i-permanently-set-my-bashrc-changes) - This didn't apply to me but could help someone else.

[Second Search](https://stackoverflow.com/questions/51876792/why-must-i-source-bashrc-every-time-i-open-terminal-for-aliases-to-work) - This one mostly answered my question.

___________________________________________

Quick look:

A typical install of OS won't create a .bash_profile for you. When you want to run functions from your command line, this is a must-have.

**Start up Terminal**
 1. Type cd ~/ to go to your home folder
 2. Type touch .bash_profile to create your new file.
 3. Edit .bash_profile with your favorite editor (or you can just type open -e .bash_profile to open it in TextEdit.
 4. [ -f "$HOME/.bashrc" ] && source "$HOME/.bashrc" Save it and close it

Here is what I did personally. I deleted my .bashrc file from ``~/workspace/.bashrc`` and recreated it in the ``~`` root directory. After that I used ``touch .bash_profile`` and then ``code .bash_profile`` to open it in VSC. Interestingly enough the ``.bash_profile`` created the right commands to source ``.bashrc`` permanently for me so I never used step 4. 

Hope this helps!
      -Jesse

**Comment by drteresavasquez** [comment]
This is excellent!
